backend/models/feedbackModel.js

```diff
@@ -6,9 +6,22 @@ const feedbackSchema = mongoose.Schema(
       type: String,
       required: true,
     },
+    user_id: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+    },
     task_id: {
       type: mongoose.Schema.Types.ObjectId,
+      required: true,
+    },
+    satisfaction: {
+      type: Number,
+      required: false
```

Comment:
The satisfaction should be a required value

Reply:
Fixed: Changes are reflected in recent commits